cedar-policy-validator/src/schema/namespace_def.rs

```diff
@@ -58,8 +58,8 @@ fn action_entity_type_parses() {
 /// Return true when an entity type is an action entity type. This compares the
 /// base name for the type, so this will return true for any entity type named
 /// `Action` regardless of namespaces.
-pub(crate) fn is_action_entity_type(ty: &Name) -> bool {
-    ty.basename().as_ref() == ACTION_ENTITY_TYPE
+pub(crate) fn is_action_entity_type(ty: &EntityType) -> bool {
```

Comment:
Maybe we should move this comment to the `is_action()` function, just to avoid a future world where we change the definition there and forget to update this (faraway) comment